lib/rentlinx/errors.rb

```diff
@@ -7,4 +7,37 @@ class UnexpectedAttributes < RentlinxError
 
   class InvalidTypeParam < RentlinxError
   end
+
+  class InvalidObject < RentlinxError
+    def initialize(type)
+      super("The given #{type} is missing required attributes and cannot be posted.")
```

Comment:
It would be nice to pass the object to the error so that we can specify which required attributes are missing.